app/src/main/java/co/co/nimblehq/showcases/poly/place/ui/screens/restaurant/nearby/NearbyRestaurantsScreen.kt

```diff
@@ -0,0 +1,204 @@
+                R.drawable.im_restaurant_mock
+            )!!.toBitmap(width = 600, height = 600),
+            name = "Dragon X",
+            address = "111, Dr. Strange street",
+            distance = "50 m"
+        )
+    )
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .paint(
+                painter = painterResource(id = R.drawable.bg_nearby_restaurants),
+                contentScale = ContentScale.FillBounds
+            )
+    ) {
+        NearbyRestaurantsToolbar()
+        NearbyRestaurantList(mockRestaurants)
+    }
+    Timber.d("Result : $uiModels")
+}
+
+@Composable
+private fun NearbyRestaurantsToolbar() {
+    ConstraintLayout(modifier = Modifier.fillMaxWidth()) {
+        val (title, iconEnd, divider) = createRefs()
+
+        val spacingNormal = dimensions.spacingNormal
+        val spacingLarge = dimensions.spacingLarge
+        val spacingHuge = dimensions.spacingHuge
+
+        Text(
+            modifier = Modifier
+                .constrainAs(title) {
+                    top.linkTo(parent.top, spacingHuge)
+                    start.linkTo(parent.start, spacingNormal)
+                },
+            text = stringResource(R.string.nearby_restaurants_title),
+            color = Color.White,
+            fontSize = dimensions.textSizeToolbar
+        )
+        Icon(
+            modifier = Modifier
+                .size(dimensions.iconSizeToolbar)
+                .constrainAs(iconEnd) {
+                    end.linkTo(parent.end, spacingLarge)
+                    centerVerticallyTo(title)
+                },
+            painter = painterResource(id = R.drawable.ic_target),
+            contentDescription = null,
+            tint = BlueDenim
+        )
+        Divider(
+            modifier = Modifier
+                .padding(
+                    horizontal = dimensions.spacingNormal,
+                    vertical = dimensions.spacingSmall
+                )
+                .constrainAs(divider) {
+                    top.linkTo(title.bottom)
+                    centerHorizontallyTo(parent)
+                },
+            color = White20
+        )
+    }
+}
+
+@Composable
+private fun NearbyRestaurantList(
+    restaurants: List<Restaurant>
+) {
+    LazyVerticalGrid(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(
+                start = dimensions.spacingNormal,
+                end = dimensions.spacingNormal,
+                top = dimensions.spacingNormal,
+                bottom = dimensions.spacingHuge
+            ),
+        verticalArrangement = Arrangement.spacedBy(dimensions.spacingNormal),
+        horizontalArrangement = Arrangement.spacedBy(dimensions.spacingNormal),
+        columns = GridCells.Fixed(RESTAURANTS_GRID_COLUMNS)
+    ) {
+        itemsIndexed(
+            items = restaurants,
+            key = { _, item -> item.id }
+        ) { _, item ->
+            NearbyRestaurantItem(item)
+        }
```

Comment:
It seems like we don't need the index, right? Can we switch using a simple `items` instead?
```suggestion
        items(
            items = restaurants,
            key = { it.id }
        ) { item ->
            NearbyRestaurantItem(item)
        }
```